sklearn/metrics/tests/test_ranking.py

```diff
@@ -126,14 +126,16 @@ def _average_precision(y_true, y_score):
 def test_roc_curve():
     # Test Area under Receiver Operating Characteristic (ROC) curve
     y_true, _, probas_pred = make_prediction(binary=True)
-
-    fpr, tpr, thresholds = roc_curve(y_true, probas_pred)
-    roc_auc = auc(fpr, tpr)
     expected_auc = _auc(y_true, probas_pred)
-    assert_array_almost_equal(roc_auc, expected_auc, decimal=2)
-    assert_almost_equal(roc_auc, roc_auc_score(y_true, probas_pred))
-    assert_equal(fpr.shape, tpr.shape)
-    assert_equal(fpr.shape, thresholds.shape)
+
+    for drop in [True, False]:
```

Comment:
This test tells us nothing if that particular case didn't need to drop anything. Ensure that something is actually dropped, and then hopefully asserting the equivalence of AUC should suffice as a test.